pldoc/static/js/ui.js

```diff
@@ -20,6 +20,12 @@ define([
 
         setupHtml: function() {
             $('html').removeClass('no-js');
+            $('#pldoc-nav-elements .nav-list-item .pldoc-link').each(function (index, element) {
+                if ( $( element ).attr('href') === window.location.pathname ) {
+                  $( element ).addClass('is-current');
```

Comment:
One last nit: it would be more performant to create a ``` var $el = $( element ); ``` at the start of each loop instead of creating the same jQuery element object twice.